content/browser/webui/brave_shared_resources_data_source.cc

```diff
@@ -162,7 +167,12 @@ BraveSharedResourcesDataSource::GetAccessControlAllowOriginForOrigin(
   // back.
   std::string allowed_origin_prefix = content::kChromeUIScheme;
   allowed_origin_prefix += "://";
+  std::string allowed_untrusted_origin_prefix =
+      content::kChromeUIUntrustedScheme;
+  allowed_untrusted_origin_prefix += "://";
   if (!base::StartsWith(origin, allowed_origin_prefix,
+                        base::CompareCase::SENSITIVE) &&
+      !base::StartsWith(origin, allowed_untrusted_origin_prefix,
                         base::CompareCase::SENSITIVE)) {
     return "null";
```

Comment:
actually `null` is unsafe unless you are intentionally granting access to non-hierarchical URL schemes. if not, you should just omit this header. https://w3c.github.io/webappsec-cors-for-developers/#avoid-returning-access-control-allow-origin-null

Reply:
ok, I've updated this to omit the header